nbconvert/exporters/tests/test_pdf.py

```diff
@@ -37,4 +37,6 @@ def test_export(self):
             (output, resources) = self.exporter_class(latex_count=1).from_filename(newpath)
```

Comment:
I'd feel better if this test would run LaTeX with a bibfile since that can also introduce artifacts and should be tested as part of the smoke test. If this turns out to be too big of a change, then we can move this into a new PR & make a new issue. But, could you give that a shot?

Reply:
I can try to do that. Do you think it makes sense to copy these examples files into the repo for the test? https://github.com/jupyter/nbconvert-examples/tree/master/citations I can't seem to find an existing example notebook in this repo.

Reply:
I think that those might be out of date — given that they were last updated over 3 years ago. 

I'm now thinking that there may be no built-in way to do this… which is… odd. 

We may need to look into the LaTeX templates to see if we need to include a new bibliography area and traitlet to get that to work. That's starting to sound like a separate PR though.

Could you try to figure out if any of the other code related to this is dead? If you just include `assert False` in places it should trigger failures in the tests if those parts of the code are actually being tested.

Reply:
Looking at the [code coverage](https://codecov.io/gh/jupyter/nbconvert/src/2210f10d96295906c330718c60b8189d8cded77a/nbconvert/exporters/pdf.py), it seems like the only lines not executed in the PDF exporting are the win32 related, error related, or for generating the bib.

I am not sure exactly when the code became dead, if there are other files that are affected.

Reply:
Code coverage actually doesn't get exactly what I was looking for. My ultimate aim was to know which tests were using the bibliography functionality to see _how_ it was being used. I can just do this myself though…

Reply:
Alright… so I'm fairly sure that right now that this code is only being run in the tests for the nbconvertapp, which are a gigantic pain to debug and investigate because they're hidden behind a `subprocess.Popen` call to `nbconvert` as part of the test… which means that pdb just causes everything to fail.

Reply:
Regardless my investigations left me believing that currently there is nothing that's properly testing the bibliography support… so we should look into that, but that's a separate PR.

Reply:
Sounds good! I don't know much about bib support and haven't used it.